main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-bulletin-api/api"
+	"github.com/ONSdigital/dp-bulletin-api/config"
+	"os"
+	"os/signal"
+	"time"
+
+	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+
+	"github.com/ONSdigital/go-ns/server"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/mux"
+)
+
+var (
+	// BuildTime represents the time in which the service was built
+	BuildTime string
+	// GitCommit represents the commit (SHA-1) hash of the service that is running
+	GitCommit string
+	// Version represents the version of the service that is running
+	Version string
+)
+
+func main() {
+	if err := run(os.Args); err != nil {
+		log.Error(err)
+		os.Exit(1)
+	}
+}
+
+// run the application
+func run(args []string) error {
```

Comment:
As per this comment on an unrelated PR (https://github.com/ONSdigital/dp-frontend-renderer/pull/347#discussion_r383443000) I intend to move `run()` to a separate `service` package and to update the PR.